File: src/main/java/team4/footwithme/resevation/repository/CustomMercenaryRepository.java
package team4.footwithme.resevation.repository;

import org.springframework.data.domain.Page;
import org.springframework.data.domain.Pageable;
import team4.footwithme.resevation.domain.Mercenary;

public interface CustomMercenaryRepository {
    Page<Mercenary> findAllToPage(Pageable pageable);
}

File: src/main/java/team4/footwithme/resevation/repository/CustomMercenaryRepository.kt
package team4.footwithme.resevation.repository

import org.springframework.data.domain.Page
import org.springframework.data.domain.Pageable
import team4.footwithme.resevation.domain.Mercenary

interface CustomMercenaryRepository {
    fun findAllToPage(pageable: Pageable): Page<Mercenary>
}
